file-formats/bestpractices.md

```diff
@@ -0,0 +1,23 @@
+# Best practices
+
+openEO Platform offers different [file formats](/file-formats) for importing and exporting of data. Depending on the data cube that your process graph creates and on your later use case, some file formats are more suitable than others.
+
+This page provides a brief overview of the most common use cases:
+
+### Raster formats
+- JPEG/PNG: data formats that are well suited for use in media/printing
+  - not georeferenced (therefore only limited usable for further analysis)
+  - limited to 3 (JPEG) or 4 (PNG) output bands (image channels)
+  - usually contains data from 1 timestamp
+- netCDF: ideal for time series data as it stores data in multi-dimensional arrays
+  - georeferenced (x/y dimensions)
+  - can store multiple bands (band dimension)
+  - can store multiple timestamps (time dimension)
+  - self-describing, portable and scalable
+- GeoTiff: 
+  - georeferenced
+  - can store multiple bands
+  - usually describes data for one timestamp
```

Comment:
I don't really agree with this one: COG+STAC is advertised by some as the final solution to publishing full multitemporal, multiband EO collections in an efficient way, and openEO is also fully trying to support STAC. Landsat is one example of a very important collection that follows this approach.
So having a best practice that hints at the opposite seems in conflict.
Suggested alternative:
"A single GeoTiff corresponds to one timestamp, but in combination with STAC, multitemporal collections are supported."

Reply:
If we speak about how we have the data/collections stored I fully agree, this page was more intended to give hints about which formats to use in `save_result` where I think non of the backends yet supports what you suggested (though I remember you had plans to implement this at some point).

We already mention [here](https://docs.openeo.cloud/federation/backends/fileformats.html#geotiff) "One timestamp per geotiff, as geotiffs do not support multiple timestamps."

I updated the text to make it hopefully more clear and less conflicting.

Btw. found this yesterday which is a really nice related read: https://radiant.earth/blog/2023/03/the-naive-origins-of-the-cloud-optimized-geotiff